src/App.js

```diff
@@ -1,9 +1,68 @@
+import CoffeeMatch from "./CoffeeMatch";
+import "./assets/aynu.png";
```

Comment:
Import the image like this:
```suggestion
import AynuImage from "./assets/aynu.png";
```